synapse/storage/databases/main/event_push_actions.py

```diff
@@ -522,6 +639,7 @@ def _get_notif_unread_count_for_user_room(
         user_id: str,
         stream_ordering: int,
         max_stream_ordering: Optional[int] = None,
+        thread_id: Optional[str] = None,
```

Comment:
> (I've updated the docstring to add the `thread_id` parameter.)
>
> *-- https://github.com/matrix-org/synapse/pull/13877#discussion_r982614240*

Was this pushed up? I'm not seeing a change to the docstring on `_get_notif_unread_count_for_user_room`